test/interface.js

```diff
@@ -69,5 +81,22 @@ describe('monitor-dog', function() {
         done();
       });
     });
+
+    describe('sockets-monitor', function () {
+      it('should expose a `start` method', function (done) {
+        var socketsMonitor = monitor.socketsMonitor;
+        expect(socketsMonitor.start).to.exist();
+        expect(typeof socketsMonitor.start).to.equal('function');
+        done();
+      });
+
+      it('should expose a `stop` method', function (done) {
+        var socketsMonitor = monitor.socketsMonitor;
+        expect(socketsMonitor.stop).to.exist();
+        expect(typeof socketsMonitor.stop).to.equal('function');
+        done();
+      });
+    });
+
```

Comment:
yes, I was going to do that. PR description says that I'm still working on tests.